persistence-cassandra/core/src/main/resources/lagom-test-embedded-cassandra.yaml

```diff
@@ -25,6 +25,8 @@ saved_caches_directory: $DIR/saved_caches
 
 hints_directory: $DIR/hints
 
+cdc_raw_directory: $DIR/cdc_raw
+
```

Comment:
Think we can remove this file and revert to the default config shipped with Akka Persistence Cassandra.